src/Tasks.UnitTests/GetSDKReference_Tests.cs

```diff
@@ -187,15 +188,19 @@ private static string CreateFakeSDKReferenceAssemblyDirectory2(out string sdkDir
     /// </summary>
     public class GetSDKReferenceFilesTestFixture : IDisposable, IClassFixture<FakeSdkStructure>
     {
+        private readonly ITestOutputHelper _output;
+
         private readonly string _sdkDirectory;
         private readonly string _sdkDirectory2;
         private readonly MockEngine.GetStringDelegate _resourceDelegate;
         private readonly GetAssemblyName _getAssemblyName = GetAssemblyName;
         private readonly GetAssemblyRuntimeVersion _getAssemblyRuntimeVersion = GetImageRuntimeVersion;
         private readonly string _cacheDirectory = Path.Combine(Path.GetTempPath(), "GetSDKReferenceFiles");
 
-        public GetSDKReferenceFilesTestFixture(FakeSdkStructure fakeSdkStructure)
+        public GetSDKReferenceFilesTestFixture(FakeSdkStructure fakeSdkStructure, ITestOutputHelper output)
```

Comment:
The xunit runner calls it (finds it by reflecting over the assembly somehow), so there are no direct callers.